specs/waku/waku-1.md

```diff
@@ -314,11 +314,11 @@ Each node SHOULD respect rate limits advertised by its peers. The number of pack
 
 ##### Light Node Field
 
-Light Node represents a boolean option. When true, the peer SHOULD NOT forward Envelopes through the Messages packet (`0x01`).
+When the `light-node` field is set to true, the host is communicating to its peers that it will not forward Envelopes through the Messages packet (`0x01`).
```

Comment:
I don't really like this change. It removed rfc2119 language.

Reply:
see above